apps/ledger-live-desktop/src/renderer/components/AccountSubHeader/AccountSubHeaderDrawer.tsx

```diff
@@ -19,7 +30,34 @@ const Title = styled(Text)`
 const Description = styled(Text)`
   font-size: 13px;
 `;
-export function AccountSubHeaderDrawer({ isOpen, closeDrawer, family, team }: Props) {
+
+type TeamLinkProps = {
+  team: string;
+  teamLink: string;
+};
+
+const TeamLink = ({ team, teamLink }: TeamLinkProps) => {
+  const { t } = useTranslation();
+  return (
+    <>
+      <Description>{t("account.subHeader.drawer.descriptionLink.integration")}</Description>{" "}
+      <Link
+        verticalAlign={"bottom"}
+        onClick={() => openURL(teamLink)}
+        justifyContent="center"
+        color={"palette.primary.main"}
+      >
+        {team}
+        <ExternalLinkIconContainer>
+          <IconExternalLink size={12} />
+        </ExternalLinkIconContainer>
+      </Link>{" "}
```

Comment:
remove the `{" "}`

Reply:
It's actually required to not have the link stuck to the surrounding works, not using padding/margin allows the renderer to break the line where needed.

Reply:
you can use a none braking space , its more native than this. 
`&nbsp;`

Reply:
Thanks for the tip!